assets/js/blocks/product-gallery/inner-blocks/product-gallery-large-image/editor.scss

```diff
@@ -0,0 +1,3 @@
+.wc-block-editor-product-gallery-large-image img {
+	max-width: 500px;
```

Comment:
This is a holdover from the legacy block. I have added it, since it prevents the Large Image and Thumbnails from overflowing the Product Gallery container in the editor:

<img width="1233" alt="Cursor_i_Single_Product_‹_Template_Part_‹_Editor_‹_productgallery_—_WordPress" src="https://github.com/woocommerce/woocommerce-blocks/assets/905781/ff3cd52f-67ad-4fd5-b72d-22551000d6da">

I'm happy with refactoring it in the future.